Compiler/SimCode/SimCodeMain.mo

```diff
@@ -662,7 +677,9 @@ algorithm
     local
       String str, newdir, newpath, resourcesDir, dirname;
       String fmutmp;
+      String guid;
       Boolean b;
+      String fileprefix;
```

Comment:
`guid` and `fileprefix` doesn't seem to be used...

Reply:
They are used in the "full" version of OMSI, but in `callTargetTemplatesFMU` we can not call the new template functions for omsi yet. I can delete them here and include them in the commit, which will add the template functions.

Reply:
Never mind 👍